lib/components/cards/index.css

```diff
@@ -0,0 +1,30 @@
+.card {
+  max-width: 16rem;
+  height: fit-content;
+  max-height: 22rem;
+  box-shadow: 0 1px 4px 1px #acacac;
+  border-radius: 4px;
+  border: 1px solid #f9f9f9;
+  position: relative;
+}
+.card-img {
+  height: 80%;
+  object-fit: fill;
+  width: 100%;
+  max-height: 14rem;
+}
+.card-content {
+  padding: 1rem;
+}
+.card-disabled {
+  background-color: #f9f9f9;
```

Comment:
Please use color variables where ever possible

Reply:
Will update that with variable.